### DISCUSSION TITLE
Turning on descriptive CSS labels for production builds

### DISCUSSION THREAD

**Discussion by conatus** [system-ui/theme-ui#1637] [Q&A]
Hey all,

Currently `theme-ui` only gives classes descriptive names in development mode.

This is because this is set as the default by Emotion's Babel plugin downstream from this.

https://github.com/emotion-js/emotion/tree/main/packages/babel-plugin#options

Do you have a steer how I could turn `autoLabel` to `always` in my Next.js project so I can get these descriptions in production?

Thanks a lot!

**Comment by conatus** [comment]
Discussion of this issue in the Emotion repository. No clues as of yet . :)

https://github.com/emotion-js/emotion/issues/756#issuecomment-816566594

**Comment by hasparus** [comment]
Are you using `sx` prop, or `@theme-ui/components`?
